sale_stock_available_to_promise_release/views/res_config_settings.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="res_config_settings_view" model="ir.ui.view">
+        <field name="model">res.config.settings</field>
+        <field
+            name="inherit_id"
+            ref="stock.res_config_settings_view_form"
+        />
+        <field name="arch" type="xml">
+            <xpath expr="//div[@id='annual_inventory_date']" position="after">
```

Comment:
There is already a section introduced by the parent module to host settings related to the release features: https://github.com/OCA/wms/blob/16.0/stock_available_to_promise_release/views/res_config_settings.xml#L13C64-L13C81
Better to place the new setting in this section.